extension/framework/framework.go

```diff
@@ -33,9 +35,28 @@ var log = logging.MustGetLogger("extest")
 func TestExtensions(c *cli.Context) {
 	buflog.SetUpDefaultLogging()
 
+	var config *util.Config
+	configFilePath := c.String("config-file")
+
+	if configFilePath != "" && !c.Bool("verbose") {
+		config = util.GetConfig()
+		err := config.ReadConfig(configFilePath)
+		if err != nil {
+			log.Error(fmt.Sprintf("Failed to read config from path %s", configFilePath))
+			os.Exit(1)
+		}
+
+		err = logg.SetUpLogging(config)
+		if err != nil {
+			log.Error(fmt.Sprintf("Failed to set up logging: %v", err))
+			os.Exit(1)
+		}
+	}
+
 	testFiles := getTestFiles(c.Args())
 
-	returnCode := RunTests(testFiles, c.Bool("verbose"))
+	//logging from config is a limited printAllLogs option
+	returnCode := RunTests(testFiles, c.Bool("verbose") || config != nil)
```

Comment:
Why enable `verbose` when there is a config? I thought they would be independent options (`verbose` meaning "print logs for passing tests too"; and the config specifying the log levels for what gets printed, regardless if it's only logs for failing tests).

A config option for verbosity can be added, if it's needed.

Or, if we don't want `verbose` being disabled by default, this can be changed.